backends/platform/iphone/iphone_video.mm

```diff
@@ -158,31 +166,74 @@ - (void)createContext {
 	}
 }
 
+- (void)setupGestureRecognizers {
+	UISwipeGestureRecognizer *swipeRight = [[UISwipeGestureRecognizer alloc] initWithTarget:self action:@selector(twoFingersSwipeRight:)];
```

Comment:
This is something I feared, indeed… The major issue is that the handling of screen orientation has changed with iOS 7, and writing something which works on both cases is very complex.